pkg/client/didexchange/client_test.go

```diff
@@ -500,93 +500,21 @@ func TestAcceptExchangeRequest(t *testing.T) {
 	require.Contains(t, err.Error(), "did exchange client - accept exchange request:")
 }
 
-func TestServiceEventError(t *testing.T) {
-	didExSvc := mockprotocol.MockDIDExchangeSvc{
-		ProtocolName:           didexchange.DIDExchange,
-		RegisterActionEventErr: errors.New("action event registration failed"),
-		RegisterMsgEventErr:    errors.New("msg event registration failed"),
-	}
-
-	// register action event on service throws error
-	_, err := New(&mockprovider.Provider{StorageProviderValue: mockstore.NewMockStoreProvider(),
-		ServiceValue: &didExSvc})
-	require.Error(t, err)
-	require.Contains(t, err.Error(), "didexchange action event "+
```

Comment:
Same as https://github.com/hyperledger/aries-framework-go/pull/658#discussion_r342085399. I understand we are reusing the events related code from service, but API surface for consumers hasn't changed. Just curious about removal of these test cases.

Reply:
previously the function `New` did registration for  Action and Msg event. I have removed it,  no code no test

https://github.com/hyperledger/aries-framework-go/pull/658/files/3698253ec6234300a5133a5fec475986dac2484e#diff-427522b353a28b1ec350c29c79d387b0L87-L97